apps/core/scenes/results/Results.js

```diff
@@ -181,6 +192,11 @@ const styles = StyleSheet.create({
     },
     justifyContent: 'flex-start',
   },
+  searchFormContainer: {
+    paddingHorizontal: 125,
+    paddingVertical: 20,
+    minWidth: 500,
```

Comment:
Isn't it pity to set minWidth when design is responsive?

Reply:
Yes! Looks pretty ugly to me, but if not that happens: 

<img width="316" alt="Screenshot 2019-04-16 at 16 15 17" src="https://user-images.githubusercontent.com/11021461/56217069-dea61a80-6062-11e9-9dba-9158478c4129.png">

Any idea on how to do that better? @jvaclavik

Reply:
True, we can also save some space by defining some smaller `paddingHorizontal` value for smaller layouts.

<img width="517" alt="Screenshot 2019-04-16 at 16 17 08" src="https://user-images.githubusercontent.com/2660330/56217280-3fcdee00-6063-11e9-8f78-d9fecf813f03.png">